src/app/home-panel/home-panel.component.less

```diff
@@ -36,11 +36,13 @@ mat-icon {
 .furnitureSlots {
   display: flex;
   flex-wrap: wrap;
+  gap: 3px;
+  margin-bottom: 2px;
 }
 
 .itemSlot {
   width: 50px;
-  height: 36px;
+  height: 50px;
```

Comment:
Minor, is there a specific reason why Furniture appears to be ~8 pixels smaller than equipment?

Reply:
It was only because I found furniture to look better without the excess space.

Reply:
I also found the panel having square icons to be much more appealing.